net/mosquitto/Makefile

```diff
@@ -144,7 +144,7 @@ endef
 define Package/libmosquittopp
     SECTION:=libs
     CATEGORY:=Libraries
-    DEPENDS:=+libmosquitto-ssl +libstdcpp
+    DEPENDS:=+libmosquitto +libmosquitto-ssl +libstdcpp
```

Comment:
I think you should actually try changing this to just DEPENDS:=libmosquitto (without the +)

libmosquitto is a virtual package now, to allow the multiple providers, so the + selection probably doesn't work?

Reply:
those are all separate issues.  are you saying that libwebsockets-cyassl doesn't compile? file that and we can work on that.

Did you try my suggested change for libmosquittopp?